src/cljs/lambdacd/logic.cljs

```diff
@@ -10,25 +10,33 @@
 
 (defn start-ticker []
   (go-loop []
-           (re-frame/dispatch [::tick])
-           (async/<! (async/timeout poll-frequency))
-           (recur)))
+           (let [update-in-progress? @(re-frame/subscribe [::db/update-in-progress?])]
+             (when-not update-in-progress?
+               (re-frame/dispatch [::tick])
+               (async/<! (async/timeout poll-frequency))
+               (recur)))))
```

Comment:
The timeout and recur should probably be outside the the `when-not`. 

It looks like at the moment, this breaks the update mechanism when requests would overlap. I reproduced it by setting the network to "slow 3g" in the chrome dev-tools. After I do that, the requests take longer than a second and the updates just stop.

Reply:
@flosell nice that you have noticed that, you totally right, i reproduced it too, fixed